<div style="background:linear-gradient(135deg,#042f2e 0%,#0f766e 55%,#14b8a6 100%);border-radius:18px;padding:34px 30px;color:#fff;font-family:Inter,Segoe UI,sans-serif">
  <div style="font-size:12px;letter-spacing:3px;color:#99f6e4;font-weight:700;text-transform:uppercase">Chapter 38 · Solutions</div>
  <div style="font-size:34px;font-weight:900;line-height:1.1;margin:10px 0 6px">Continuous Distributions, Worked Answers ✅</div>
  <div style="font-size:15px;color:#d1fae5;max-width:720px;line-height:1.6">Full solutions to the five challenges. Try them yourself first, then compare.</div>
  <div style="margin-top:16px;font-size:13px;color:#a7f3d0">Statistics, Data Science and AI: A Visual Handbook · John Fisher · 2026</div>
</div>

## ⚙️ Setup

In [1]:
import numpy as np
from scipy import stats
rng = np.random.default_rng(381)
print("ready")

ready


<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0d9488;letter-spacing:1px">CHALLENGE 1 · UNIFORM</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Flat probability</div>
<div style="color:#4a5578;margin-top:6px">A bus is equally likely to arrive at any minute in a 10-minute window, X ~ Uniform(0, 10). Find P(X < 3) and the mean wait.</div>
</div>

In [2]:
print(f"P(X < 3)  = 3/10 = {stats.uniform.cdf(3, 0, 10):.3f}")
print(f"mean      = (a+b)/2 = {stats.uniform.mean(0, 10):.1f}")

P(X < 3)  = 3/10 = 0.300
mean      = (a+b)/2 = 5.0


**Answer:** for a uniform, probability is proportional to length, so P(X < 3) = 3/10 = **0.30**, and the mean is the midpoint (0 + 10)/2 = **5** minutes.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0891b2;letter-spacing:1px">CHALLENGE 2 · EXPONENTIAL</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Waiting times</div>
<div style="color:#4a5578;margin-top:6px">Buses arrive on average every 10 minutes (exponential). Find P(wait > 15 min) and P(wait < 5 min).</div>
</div>

In [3]:
scale = 10        # mean = 1/lambda = 10
print(f"P(wait > 15) = e^(-15/10) = {1 - stats.expon.cdf(15, scale=scale):.4f}")
print(f"P(wait < 5)  = 1 - e^(-5/10) = {stats.expon.cdf(5, scale=scale):.4f}")

P(wait > 15) = e^(-15/10) = 0.2231
P(wait < 5)  = 1 - e^(-5/10) = 0.3935


**Answer:** P(wait > 15) = e&#8315;&#185;&#183;&#8309; &approx; **0.223**, and P(wait < 5) = 1 &minus; e&#8315;&#8304;&#183;&#8309; &approx; **0.393**. The exponential has a long right tail, so long waits, while less likely, are far from rare.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0d9488;letter-spacing:1px">CHALLENGE 3 · NORMAL</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Probabilities under the bell</div>
<div style="color:#4a5578;margin-top:6px">IQ scores are modeled as X ~ Normal(100, 15). Find P(X < 115) and P(85 < X < 115).</div>
</div>

In [4]:
mu, sd = 100, 15
print(f"P(X < 115)        = {stats.norm.cdf(115, mu, sd):.4f}")
p_mid = stats.norm.cdf(115, mu, sd) - stats.norm.cdf(85, mu, sd)
print(f"P(85 < X < 115)   = {p_mid:.4f}")

P(X < 115)        = 0.8413
P(85 < X < 115)   = 0.6827


**Answer:** 115 is one standard deviation above the mean, so P(X < 115) = &Phi;(1) &approx; **0.8413**. The range 85 to 115 is exactly &#956; &plusmn; 1&#963;, so P(85 < X < 115) &approx; **0.6827**, the famous 68% from the empirical rule (Chapter 39).

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#db2777;letter-spacing:1px">CHALLENGE 4 · QUANTILES</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Reading the CDF backward</div>
<div style="color:#4a5578;margin-top:6px">For the same IQ distribution N(100, 15), find the 90th percentile, the score that 90% of people fall below.</div>
</div>

In [5]:
mu, sd = 100, 15
p90 = stats.norm.ppf(0.90, mu, sd)
print(f"90th percentile = {p90:.2f}")

90th percentile = 119.22


**Answer:** the 90th percentile is &#956; + z&#8320;.&#8313;&#8320;&#183;&#963; = 100 + 1.2816(15) &approx; **119.2**. The percent-point function (inverse CDF) turns a probability back into a value, the reverse of asking "what fraction is below x".

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0f766e;letter-spacing:1px">CHALLENGE 5 · FIT A GAUSSIAN</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Maximum likelihood</div>
<div style="color:#4a5578;margin-top:6px">Given a sample of measurements, estimate the parameters of a normal distribution by maximum likelihood.</div>
</div>

In [6]:
sample = rng.normal(50, 12, size=300)
mu_hat, sigma_hat = sample.mean(), sample.std()
print(f"MLE mu_hat    = {mu_hat:.2f}")
print(f"MLE sigma_hat = {sigma_hat:.2f}")
print("Assuming Gaussian errors, minimizing squared error == maximizing this likelihood.")

MLE mu_hat    = 48.98
MLE sigma_hat = 12.52
Assuming Gaussian errors, minimizing squared error == maximizing this likelihood.


**Answer:** the maximum-likelihood estimates are simply the sample mean and sample standard deviation (here near 50 and 12, the values we generated from). The deeper point: assuming normally distributed errors makes **least-squares fitting the maximum-likelihood choice**, which is why squared-error loss is everywhere in regression and deep learning.

---
<div style="background:#ffffff;border:1px solid #e6e9f2;border-radius:16px;padding:22px 26px;font-family:Inter,sans-serif">
<div style="font-size:19px;font-weight:800;color:#1a2138">🎉 Nicely done</div>
<div style="color:#4a5578;line-height:1.8;margin-top:8px">You worked with the uniform, exponential, and normal, read probabilities and percentiles off the CDF, and fit a Gaussian by maximum likelihood. Areas, not heights, are probabilities, and the normal is the distribution you will reach for most often.</div>
</div>

<div style="text-align:center;color:#8b94b3;font-size:12px;margin-top:16px">Statistics, Data Science and AI: A Visual Handbook · © 2026 John Fisher</div>